packages/utils/videoPlayer.ts

```diff
@@ -0,0 +1,29 @@
+//nDuration: seconds
+export const paddingZero = (n: number): string => {
+  if (n < 10) return `0${n}`;
+  return `${n}`;
+};
+export const durationToString = (nDuration: number): string => {
```

Comment:
we can use `moment` format

Reply:
Function removed since we have https://github.com/TERITORI/teritori-dapp/blob/a9a64caec2aae96b45ca9031fe62182698c4a2ba/packages/utils/mediaPlayer.ts#L1

Reply:
I remove this file since no exported const from it are used